### DISCUSSION TITLE
StateList: Sortieren

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#2189] [Q&A]
Hallo,

gibt es die Möglichkeit, eine StateList mit mehreren Geräten zu sortieren?

Danke

**Comment by mcuiobroker** [comment]
Doku Suche?

**Comment by AFPass** [comment]
Hier https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/module/statelist finde ich nichts dazu.
Übersehe ich etwas?

**Comment by mcuiobroker** [comment]
Es reicht Statelist sort in die Suche einzugeben. 

https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/system-effektprog./v3-systemprogramme/statelistsort

**Reply by AFPass** [reply]
Uff. Ok. Danke

**Comment by Zefau** [comment]
Dafür fehlt ein Feature Request für den jarvis "Standard" oder?

**Reply by mcuiobroker** [reply]
Ja, mach direkt einen auf. Bin unterwegs.

**Comment by AFPass** [comment]
#2190
